### PR TITLE
Fix broken links in documentation

### PR DESCRIPTION
**Pull request by dpgraham**
(connected to: https://github.com/appium/appium/issues/11014)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Check out our [Getting Started](/docs/en/about-appium/getting-started.md) guide
 to get going with Appium.
 
 There is also sample code that contains [many examples of tests in a variety
-of different languages](https://github.com/appium/appium/sample-code)!
+of different languages](https://github.com/appium/sample-code)!
```
Comment (imurchie) [inline README.md]: This should go to `https://github.com/appium/appium/tree/master/sample-code`

File: docs/en/about-appium/getting-started.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ additional dependencies.
 
 (Chances are, you'll eventually want to automate something other than Android
 using something other than JavaScript. In that case, check out our
-[sample-code](https://github.com/appium/appium/sample-code), which has code
+[sample-code](https://github.com/appium/sample-code), which has code
```
Comment (imurchie) [inline docs/en/about-appium/getting-started.md]: Same.

File: docs/en/about-appium/getting-started.md [hunk 3]
```diff
@@ -251,7 +251,7 @@ We've only scratched the surface of what you can do with Appium. Check out
 these resources to help you on your journey:
 
 - The Appium [command reference](https://appium.io/docs/en/commands/status/) - learn about what commands are available, how to use them with specific client libraries, etc...
-- The [sample-code](https://github.com/appium/appium/sample-code) repository, where lots more code samples are available
+- The [sample-code](https://github.com/appium/sample-code) repository, where lots more code samples are available
```
Comment (imurchie) [inline docs/en/about-appium/getting-started.md]: https://github.com/appium/appium/tree/master/sample-code

Comment (dpgraham) [reply docs/en/about-appium/getting-started.md]: Right, the update sample-code

**Pull request merged**